man/systemd.resource-control.xml

```diff
@@ -691,6 +691,35 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>BPFProgram=<replaceable>BPF_ATTACH_TYPE</replaceable><constant>:</constant><replaceable>BPF_FS_PROGRAM_PATH</replaceable></varname></term>
+        <listitem>
+          <para>Add a custom cgroup BPF program.</para>
+
+          <para><varname>BPFProgram=</varname> allows attaching cgroup-bpf hooks to a systemd unit including
+          egress and ingress attachment points also exposed to <varname>IP(Egress|Ingress)FilterPath=</varname>.
+          Cgroup-bpf hooks in the form of BPF programs loaded to the BPF file system are attached with cgroup-bpf attach
+          flags determined by the unit. For details about attachment types and flags see <ulink
+          url="https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h"/></para>
+
+          <para>The specification of BPF program consists of a <replaceable>BPF_ATTACH_TYPE</replaceable> followed by a
+          <replaceable>BPF_FS_PROGRAM_PATH</replaceable> with <literal>:</literal> as the separator:
+          <replaceable>BPF_ATTACH_TYPE</replaceable><constant>:</constant><replaceable>BPF_FS_PROGRAM_PATH</replaceable>. <varname>BPF_ATTACH_TYPE</varname>
+          is the accustomed string name of BPF attach type also used in <command>bpftool</command>. <varname>BPF_ATTACH_TYPE</varname> can be one of
+          <constant>egress</constant>, <constant>ingress</constant>, <constant>sock_create</constant>, <constant>sock_ops</constant>,
+          <constant>device</constant>, <constant>bind(4|6)</constant>, <constant>connect(4|6)</constant>, <constant>post_bind(4|6)</constant>,
+          <constant>sendmsg(4|6)</constant>, <constant>sysctl</constant>, <constant>recvmsg(4|6)</constant>, <constant>(get|set)sockopt</constant>.</para>
+
+          <para>If an option with the same <replaceable>BPF_FS_PROGRAM_PATH</replaceable> and <replaceable>BPF_ATTACH_TYPE</replaceable> is passed multiple
+          times, only one program will be attached to cgroup hook. Similarly, if a program with a given BPF file system path and
```

Comment:
This should specify which one is used if defined multiple times - first or last?